meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -49,6 +49,10 @@ MBL_DEFAULT_FS_FREQ ?= "0"
 # force the fsck to run.
 MBL_DEFAULT_FS_PASSNO ?= "2"
 
+# Default device name (file name in /dev) for the storage device on which
+# partitions live.
+MBL_DEFAULT_DEVICE_NAME ?= "${MBL_WKS_STORAGE_DEVICE_NAME}"
```

Comment:
Shouldn't we add `STORAGE` word in this variable name?

Reply:
No...but yes.

The `MBL_DEFAULT_` variables in this file all follow the naming convention `MBL_DEFAULT_` + suffix of partition specific variables for which this is the default. E.g. `MBL_DEFAULT_FS_PASSNO` is the default value for `MBL_FACTORY_CONFIG_FS_PASSNO`, `MBL_HOME_FS_PASSNO`, etc. The `_process_var_with_default` function relies on this convention (which I admit isn't very nice).

The partition specific variables don't really want `STORAGE` in their names - it's obvious that they relate to storage because they relate to partitions. `_DEVICE_NAME` seems like a reasonable suffix for the partition specific variables...

Having said that, the `MBL_DEFAULT_` prefix convention for these variables is a bit rubbish, especially since the names make their way into every recipe. `MBL_DEFAULT_DEVICE_NAME` is a particularly bad example.

How about if we change the `MBL_DEFAULT_` prefix for all of these variables to `MBL_PARTITIONS_DEFAULT_`?  So we'd end up with `MBL_PARTITIONS_DEFAULT_DEVICE_NAME`, `MBL_PARTITIONS_DEFAULT_FS_PASSNO` etc.

Reply:
I much prefer the `MBL_PARTITIONS_DEFAULT_` prefix. It makes more clear the usage of the this set of variables.
If you decide to go into this path, we need to check if the user documentation needs to be updated as well.

Reply:
The user docs don't mention the `MBL_DEFAULT` variables.

Reply:
https://github.com/ARMmbed/meta-mbl/pull/797/commits/c06c592c16b4be984ed41d2e307a6a95124e201d